Test/git-issues/git-issue-327cir2.dfy.expect

```diff
@@ -1,2 +1,2 @@
-git-issue-327cir2.dfy(6,7): Error: module definition contains a cycle (note: parent modules implicitly depend on submodules): D -> E -> E -> D
+git-issue-327cir2.dfy(6,7): Error: module definition contains a cycle (note: parent modules implicitly depend on submodules): D -> E -> E -> D -> D
```

Comment:
Just a thought for a future improvement: it already feels weird to have `E` listed twice, once for the `import E` statement and once for the actual `module E` definition. Perhaps we could omit one of them, or render one as `(import E)` instead of just `E`.

Reply:
Good call-out. This looks confusing. I updated the error message to try to make this clearer.